config/example/config.txt

```diff
@@ -390,8 +385,8 @@ EXTRA_SPACE_RUIN_LEVELS_MAX 8
 ## Uncomment this to shut down the world any time it would normally reboot
 #SHUTDOWN_ON_REBOOT
 ## A command to run prior to the world shutting down, only used if the above option is enabled
-## This default value will kill Dream Daemon on Windows machines
-#SHUTDOWN_SHELL_COMMAND taskkill /f /im dreamdaemon.exe
+## This default value will kill Dream Daemon on Windows machines.
+SHUTDOWN_SHELL_COMMAND taskkill /f /im dreamdaemon.exe
```

Comment:
Recomment this. Even though the above option is disabled, its better to have both disabled.